docs/understand/performance-best-practices.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Performance Best Practices
+parent: Understanding lakeFS
+description: This section suggests performance best practices to work with lakeFS.
+nav_order: 26
+has_children: false
+--- 
+# Performance Best Practices
+{: .no_toc }
+
+{% include toc.html %}
+
+## Overview
+Use this guide to achieve the best performance with lakeFS.
+
+## Avoid huge commits
+If you're having issues with slow commits, consider performing smaller commits. The more objects in the commit, the longer the commit will take to complete.
+Moreover, a commit should represent a meaningful point in your data's lifecycle, so limiting the commit size will create a more comprehensible commit history.
+As a rule of thumb, try to keep your commits smaller than 1M objects.
+
+## Avoid concurrent commits/merges
+Concurrent commits/merges on the same branch may result in a race and may need a retry if one fails.
+Therefore, it is suggested to avoid committing/merging on the same branch from multiple sources concurrently.
+
+## Use zero-copy import
+To import object into lakeFS, either a single time or regularly, lakeFS offers a [zero-copy import](https://docs.lakefs.io/setup/import.html#zero-copy-import) feature.
+Use this feature to import a large number of objects to lakeFS, instead of simply copying them into your repository.
+This feature will create a reference to the existing objects on your bucket and avoid the copy.
+
+## Read data using the commit ID
+In cases where you are only interested in reading committed data, use a commit ID (or a tag ID) in your path (e.g: `lakefs://repo/a1b2c3`).
```

Comment:
Please also include the special notation we used for reading committed data from a branch. See my previous comment.

Reply:
I checked and we don't support @ anymore

Reply:
https://github.com/treeverse/lakeFS/blob/08a3e8798c3616b4d1bbef3d675d2df9ac27f56e/pkg/graveler/ref/resolve_ref.go#L55 ?